### PR TITLE
mobile first fixes and meeting update

### PR DESCRIPTION
**Pull request by alexandradm**
center some items, add #snaprt storify

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,20 +86,20 @@
         <br>
 
 	     <section class="row text-center">	
-            <div class=" col-md-2 col-sm-6">
-                <img class="img-responsive" src="{{ site.baseurl }}/img/ucr.jpg" alt="ucr" />
+            <div class="col-md-2 col-sm-6">
+                <center><img class="img-responsive" src="{{ site.baseurl }}/img/ucr.jpg" alt="ucr" /></center>
```
Comment (edsu) [inline index.html]: Huh, I didn't even know &lt;center&gt; was a tag -- although it [looks](https://www.w3schools.com/tags/tag_center.asp) like it's deprecated now?

**Comment by edsu** [comment]
It's a bit weird that we ended up with a conflict on this. I think perhaps a modification was made that you hadn't merged in? I can resolve it thought I think.

**Comment by alexandradm** [comment]
just updated PR with new center handling and hopefully resolved conflict

**Comment by edsu** [comment]
Seems the conflict is still there. Github can be kinda persnickety about this kind of thing. I think I can sort it out though

**Comment by alexandradm** [comment]
yeah its getting grumpy with me so ill just let you fix it and then pull down :)

**Pull request merged**